docker-compose.yml

```diff
@@ -1,23 +1,27 @@
-version: '3.1'
+version: '2'
```

Comment:
Let's change to docker compose version 3.8 as well.

Reply:
Since version 3 is for docker swarm mode and removes several useful options v2 had, I stuck with v2 (while v1 is deprecated, v2 is still supported). I guess it can be switched to version 3 without issue since the compose file no longer uses any options that are only in v2 (like volumes_from and resource limits)

Reply:
Fair enough. Users should modify this file anyways.